tempodb/backend/util/util.go

```diff
@@ -3,35 +3,37 @@ package util
 import (
 	"os"
 	"path"
+	"strconv"
 
 	"github.com/google/uuid"
 )
 
 func MetaFileName(blockID uuid.UUID, tenantID string) string {
-	return path.Join(rootPath(blockID, tenantID), "meta.json")
+	return path.Join(RootPath(blockID, tenantID), "meta.json")
 }
 
-func BloomFileName(blockID uuid.UUID, tenantID string) string {
-	return path.Join(rootPath(blockID, tenantID), "bloom")
+func BloomFileName(blockID uuid.UUID, tenantID string, bloomShard int) string {
+	return path.Join(RootPath(blockID, tenantID), "bloom-"+strconv.Itoa(bloomShard))
```

Comment:
Should we consider backwards compatibility here?  I.e. reading the existing "bloom" file if present.

Reply:
I think for the time being we're fine with breaking changes.  There will definitely come a point when we will make sure storage is always backwards compatible.